content/Assets/Styles/components/button/_variables.scss

```diff
@@ -3,22 +3,14 @@
    ========================================================================== */
 
 :root {
-    --buttonAfterHoverWipeAmount: 100%;
-    --buttonAfterOpacity: 0.1;
-    --buttonAfterTransformRotate: 100deg;
-    --buttonBorderRadius: 0;
-    --buttonBorderThickness: 0.1rem;
-    --buttonFontSize: var(--fontSizeSmall);
+    --buttonBorderRadius: var(--borderRadius);
+    --buttonBorderThickness: var(--borderWidth);
+    --buttonDisabledColor: var(--colorGrey);
+    --buttonDisabledTextColor: var(--colorGreyLight);
```

Comment:
If we do that we'll end up with a hover state on disabled buttons still, which might be a bit misleading? e.g if a button has a colour change on hover (which is our standard behaviour) and the only thing we do to disabled ones is add opacity, the colour change will still happen, at the disabled opacity. 

Is that something we'd be okay with?

Reply:
No, you're right. We'd want to avoid the hover state as that is confusing.

Happy for you to keep the colour variables in there, we just need to make sure the following is covered:

- disabled state colours match the active state colours
- disabled state also has an opacity
- disabled state has no hover state (at least not visibly)